apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -77,7 +77,7 @@ type ContourDeploymentSpec struct {
 type ContourSettings struct {
 	// Deprecated: Use `DeploymentSettings.Replicas` instead.
 	//
-	// Replicas is the desired number of Contour replicas. If if unset,
+	// Replicas is the desired number of Contour replicas. If unset,
```

Comment:
You will need to run `make generate` and commit the results to reflect this change in the CRD YAML and get CI to pass.